### PR TITLE
a11y: POC for Table resizable columns with keyboard 

### LINKED ISSUE #26739: feat: Table - Column resizing a11y
Currently there is no way the columns can be resized with a keyboard only. 

Build an example how it can be done and consider options to make this built-in for DataGrid.

### PR DESCRIPTION
**Pull request by george-cz**
## Related Issue(s)

- Fixes #26739

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/4af9156bc869948b12610689f15b2a778aa690ca'>4af9156bc869948b12610689f15b2a778aa690ca</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Avatar-mount_base.html">1331</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Avatar-mount.html">1302</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Button-mount_base.html">931</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Button-mount.html">923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Field</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Field-mount_base.html">2076</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Field-mount.html">2013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProvider-mount_base.html">1506</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProvider-mount.html">1549</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-mount_base.html">636</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-mount.html">634</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">528</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">591</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">613</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">638</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/InfoButton-mount_base.html">570</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/InfoButton-mount.html">564</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/MakeStyles-mount_base.html">1883</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/MakeStyles-mount.html">1807</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Persona-mount_base.html">3057</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/Persona-mount.html">2888</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/SpinButton-mount_base.html">2492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26873/perf-test-react-components/SpinButton-mount.html">2496</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

**Comment by fabricteam** [comment]
## Perf Analysis (`@fluentui/react-northstar`)
:warning: No perf measurements available
<div id="perfComment9424"/>

**Comment by fabricteam** [comment]
## 🕵 FluentUI-v0 No visual regressions between this PR and main
<div id="vrtCommentFluentUI-v0"/>

File: change/@fluentui-react-table-c158e951-e31e-483d-ac7e-1cb58e6c0a2d.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-table-c158e951-e31e-483d-ac7e-1cb58e6c0a2d.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26873&commitId=7a6c5f4455c375c8892364cd72f5490f9607b27e&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by george-cz** [comment]
Closing in favor of https://github.com/microsoft/fluentui/pull/26956 (This was just POC)

**Pull request closed without merging**